src/test/java/seedu/address/testutil/TypicalListings.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_APPLICANTS;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DESCRIPTION;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TITLE;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.listing.Listing;
+
+
+/**
+ * A utility class containing a list of {@code Person} objects to be used in tests.
+ */
+public class TypicalListings {
+
+    public static final Listing CHICKENRICEUNCLE = new ListingBuilder()
```

Comment:
Change to UPPER_SNAKE_CASE

Reply:
Updated this and also for toilet cleaner and software developer